testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/AssertionResponseAction.scala

```diff
@@ -193,13 +271,11 @@ object AssertionResponseAction extends ResponseActions {
    * @param minTimeMillis The minimum required duration in milliseconds, provided as a string.
    * @return A Try[Unit] that is a Success if the response's duration is greater than or equal to the specified minimum time, and a Failure with an AssertionException otherwise.
    */
-  private def assertResponseTimeIsAbove(response: Response, minTimeMillis: String): Try[Unit] = {
-    Try {
-      val lMinTimeMillis: Long = minTimeMillis.toLong
+  private def assertResponseTimeIsAbove(response: Response, minTimeMillis: String): Try[Unit] = Try {
+    val lMinTimeMillis: Long = minTimeMillis.toLong
```

Comment:
```suggestion
    val MinTimeMillis: Long = minTimeMillis.toLong
```
with input param named `minTimeMillisStr`?

same as with `iExpectedCode`?

Reply:
Fixed in commit 04e60c441ad8b397652ec2840da18f96300f1de6.